bin/pyactivate

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.9
```

Comment:
Unfortunately this creates as many problems at it solves! There are a bunch of Python v3.9 environments (CI included) that only have a /usr/bin/python3 binary and no /usr/bin/python3.9 binary.